### DISCUSSION TITLE
Is there any difference between 补习班 and 辅导班?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1124] [Q&A]
To me, these two terms have the same meaning: extracurricular classes. I'm asking because both terms appear in the text and I am not sure what are their differences. Therefore, I don't know what could be an accurate English translation for those terms. For the time being, I'll consider "extracurricular classes" as the English translation for both terms.

辅导班 is used here

https://github.com/rdrg109/subtitles-npcr/blob/03e602c945b676d2e060ef67ae958c569756845a/sentences/46-text-1.yaml#L27-L31

https://github.com/rdrg109/subtitles-npcr/blob/03e602c945b676d2e060ef67ae958c569756845a/sentences/46-text-1.yaml#L289-L294

补习班 is used here

https://github.com/rdrg109/subtitles-npcr/blob/03e602c945b676d2e060ef67ae958c569756845a/sentences/46-text-1.yaml#L246-L250

This is what I found in the dictionaries

现代汉语规范词典 (p. 408)
![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/d3d31339-436f-44d5-9d1c-2f1383d70e80)
现代汉语规范词典 (p. 100)
![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/49b648c8-e220-416f-b7b9-16269f931c0b)

This is the definition in [El Nuevo Libro de Chino Práctico (Spanish) (4th volume)](https://wikidata.org/wiki/Q118106264)

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/d3d4746e-eca0-4eff-8da4-2a3017967e9a)

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/95a3e64f-72b3-41fe-9051-e65107ea93cc)

**Comment by rodrigo-morales-1** [comment]
@fhxshrs  friendly ping

**Reply by fhxshrs** [reply]
@rdrg109  今天我也在想这个问题，刚好下午也问了一些其他老师，我们都觉得其实这两个词的差别不大，都是指想提高自己的成绩所以在课外进行额外的学习，我觉得“tutorial class”可以表示这个意思。至于你提到的“extracurricular classes”，我觉得它应该翻译为“课外课”，不单单指学业上的学习，更多的指一些关于兴趣爱好的课。

**Comment by rodrigo-morales-1** [comment]
> 都是指想提高自己的成绩所以在课外进行额外的学习

@fhxshrs Understood! Then, I think a suitable English translation for both 补习班 and 辅导班 could be "tutorial classes" or "supplementary classes". 

* "tutorial classes": I didn't know this word existed, but this is the definition that I found in [the article "Tutorial"](https://en.wikipedia.org/wiki/Tutorial) at Wikipedia.

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/0421b30b-3549-4705-a951-6f014d7c2c2c)

* "supplementary classes": The word "supplementary" indicate that the knowledge obtained in these classes supplement the knowledge that has been acquired previously (i.e. curricular classes in schools).  

I agree with you that "extracurricular classes" could refer to classes that are related to someone's personal hobbies, so it is not a suitable translation for those words in this context.

I feel that "supplementary class" is more known than "tutorial class", so I would choose the first one over the second one. @fhxshrs do you agree with me? (for the time being, I've chosen "supplementary classes", if you don't agree with me, please let me know)
